google-cloud-spanner-executor/src/main/java/com/google/cloud/executor/spanner/WorkerProxy.java

```diff
@@ -95,10 +98,22 @@ public static void main(String[] args) throws Exception {
     usePlainTextChannel = commandLine.hasOption(OPTION_USE_PLAIN_TEXT_CHANNEL);
     enableGrpcFaultInjector = commandLine.hasOption(OPTION_ENABLE_GRPC_FAULT_INJECTOR);
 
+    if (commandLine.hasOption(OPTION_MULTIPLEXED_SESSION_OPERATIONS_RATIO)) {
```

Comment:
Have you made sure this works when the value is null? Do we have some manual logs to validate that this is working?

Reply:
`OPTION_MULTIPLEXED_SESSION_OPERATIONS_RATIO` flag value will never be `null`. The default value passed from systest is `0.0`.